ledger-service/http-json-cli/src/main/scala/com/daml/http/OptionParser.scala

```diff
@@ -158,6 +158,12 @@ class OptionParser(getEnvVar: String => Option[String])(implicit
     .valueName(WebsocketConfig.usage)
     .text(s"Optional websocket configuration string. ${WebsocketConfig.help}")
 
+  opt[Unit]("log-request-and-response-bodies")
```

Comment:
Do we really want a separate option for this? I’d just fold this under the loglevel and turn it on for debug & trace (or just trace, I don’t feel strongly about this).

Reply:
This is a very good idea, I would be fine with this too. Need to adjust the changelog then.